Package.swift

```diff
@@ -8,7 +8,7 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/vapor/vapor.git", from: "3.0.0"),
-        .package(url: "https://github.com/SwiftyBeaver/SwiftyBeaver.git", from: "1.5.2")
+        .package(url: "https://github.com/SwiftyBeaver/SwiftyBeaver.git", from: "1.6")
```

Comment:
I don't know why but I get SB 1.6 without this change 😅... Anyway, just change the version to 1.6.0 to proceed with the merge 😄

Reply:
It's probably because `upToMajor` isn't being used... which I forgot is how SPM resolves versions ^_^